ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/rewrite/testtool/infrastructure/LedgerServices.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.rewrite.testtool.infrastructure
+
+import com.digitalasset.ledger.api.v1.admin.party_management_service.PartyManagementServiceGrpc
+import com.digitalasset.ledger.api.v1.admin.party_management_service.PartyManagementServiceGrpc.PartyManagementService
+import com.digitalasset.ledger.api.v1.command_service.CommandServiceGrpc
+import com.digitalasset.ledger.api.v1.command_service.CommandServiceGrpc.CommandService
+import com.digitalasset.ledger.api.v1.ledger_identity_service.LedgerIdentityServiceGrpc
+import com.digitalasset.ledger.api.v1.ledger_identity_service.LedgerIdentityServiceGrpc.LedgerIdentityService
+import com.digitalasset.ledger.api.v1.testing.time_service.TimeServiceGrpc
+import com.digitalasset.ledger.api.v1.testing.time_service.TimeServiceGrpc.TimeService
+import com.digitalasset.ledger.api.v1.transaction_service.TransactionServiceGrpc
+import com.digitalasset.ledger.api.v1.transaction_service.TransactionServiceGrpc.TransactionService
+import io.grpc.Channel
+
+private[infrastructure] final class LedgerServices(channel: Channel) {
+  val cmd: CommandService = CommandServiceGrpc.stub(channel)
```

Comment:
Do the abbreviations really improve readability?

Reply:
Absolutely not, I can have longer names. :slightly_smiling_face:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2282/commits/0ec23c48accfc125699f2dba0b237b4ec81483aa